packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -742,6 +742,32 @@ export default class TradeStore extends BaseStore {
             obj_new_values.contract_type = 'rise_fall_equal';
         }
 
+        switch (obj_new_values.contract_type) {
+            case 'even_odd':
+                obj_new_values.form_components = ['duration', 'amount'];
+                obj_new_values.trade_types = {
+                    DIGITEVEN: 'Digit Even',
+                    DIGITODD: 'Digit Odd',
+                };
```

Comment:
These values are already set in `processTradeParams`. We shouldn't resetting it.